src/seedu/addressbook/commands/CountCommand.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+/**
+ * Count all persons in the address book and display the result.
+ */
+public class CountCommand extends Command{
+
+    public static final String COMMAND_WORD = "count";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Count all persons in the address book and display the result.\n"
+            + "Example: " + COMMAND_WORD;
+
+
+    @Override
+    public CommandResult execute() {
+        List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+        return new CommandResult(getMessageForPersonListShownSummary(allPersons)/*, allPersons*/);
```

Comment:
Don't comment out code. We use version control precisely to go back to a previous version. Just go ahead and delete the commented out code.